### PR TITLE
[i18n] update italian

### PR DESCRIPTION
**Pull request by mymage**
Allignment of 
- cetmix_tower_server/i18n/it.po
- cetmix_tower_yaml/i18n/it.po

## Summary by CodeRabbit

- **New Features**
	- Updated Italian translations for server management roles and functionalities.
	- Added new translation entries for YAML data import and export processes, including user prompts and instructions.

- **Bug Fixes**
	- Enhanced error messages related to server management and YAML file handling for clearer user guidance.

- **Documentation**
	- Improved clarity of translation strings to better reflect functionalities and user actions.

### CODE REVIEW

**Comment by ivs-cetmix** [comment]
Hello @mymage , thank you for your contribution! Could you please reply to the comments in the PR. If they are relevant, just say something "Ok, thank you!" and apply the changes. Otherwise reply "This comment is not relevant because ..." and explain why it's not relevant.

File: cetmix_tower_server/i18n/it.po [hunk 1]
```diff
@@ -2654,15 +2743,14 @@ msgstr "Testo"
...
+"Please ensure that you want to allow those commands to be run anyway."
 msgstr ""
-"Il livello di accesso del/dei comando/i '%(command_names)s' incluso/i nel "
-"piano di volo corrente è superiore al livello di accesso del piano di volo "
-"stesso. Assicurati di voler davvero dare accesso a quel/i comando/i agli "
-"utenti del piano di volo."
+"Il livello di accesso dei comandi '%(command_names)s' inclusi nel piano di "
+"volo corrente è superiore al livello di accesso del piano di volo stesso. "
+"Assicurati di voler davvero consentire a tutti quei comandi di essere "
+"eseguiti comunque."
```
[the remark this answers is not in the document]
Comment (mymage) [reply cetmix_tower_server/i18n/it.po:2753]: Suggestion applied, thanks.

**Comment by mymage** [comment]
Hi @ivs-cetmix, I didn't know this feature, I hope I did the right thing.

**Comment by CetmixGitDrone** [comment]
On my way to merge this fine PR!
Prepared branch [14.0-dev-ocabot-merge-pr-157-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-157-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 78352a7d161ad203645efe42368f24360d42df90. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**